freqtrade/optimize/hyperopt.py

```diff
@@ -295,6 +296,50 @@ def _format_explanation_string(results, total_epochs) -> str:
                 f"{results['results_explanation']} " +
                 f"Objective: {results['loss']:.5f}")
 
+    @staticmethod
+    def print_result_table(config: dict, results: list, total_epochs: int, highlight_best: bool,
+                           print_colorized: bool) -> None:
+        """
+        Log result table
+        """
+        if not results:
+            return
+
+        trials = json_normalize(results, max_level=1)
+        trials = trials[['is_best', 'current_epoch',
+                         'results_metrics.trade_count', 'results_metrics.avg_profit',
+                         'results_metrics.total_profit', 'results_metrics.profit',
+                         'results_metrics.duration', 'loss']]
+        trials.columns = ['Best', 'Epoch', 'Trades', 'Avg profit', 'Total profit',
+                          'Profit', 'Avg duration', 'Objective']
+
+        trials['Best'] = trials['Best'].apply(lambda x: '*' if x else '')
+        trials['Objective'] = trials['Objective'].astype(str)
+
+        if print_colorized:
+            for i in range(len(trials)):
+                if trials.loc[i]['Total profit'] > 0:
+                    trials.at[i, 'Best'] = Fore.GREEN + trials.loc[i]['Best']
+                    trials.at[i, 'Objective'] = "{}{}".format(trials.loc[i]['Objective'],
+                                                              Fore.RESET)
+                    if '*' in trials.loc[i]['Best'] and highlight_best:
```

Comment:
Is it shifted one level right intentionally or by error? This changes the colorization schema (not profitable bests are now also printed in bold/bright). I would propose to keep the current colorization schema (we've chosen it from 4 or 6 variants, when it was implemented, so it's 'carefully selected' 😉 )

Reply:
Yeah it was intentional. Didn't think it makes sense to color mark things that do not make us a profit :-D changed now.